bundles/AdminBundle/Resources/public/js/pimcore/document/tree.js

```diff
@@ -1339,20 +1338,21 @@ pimcore.document.tree = Class.create({
                         },
                         keyup: function (el) {
                             pageForm.getComponent("name").setValue(el.getValue());
-                        }
+                            pageForm.getComponent("key").setValue(el.getValue());
+                        }.bind(this),
                     }
                 },{
                     xtype: "textfield",
                     itemId: "name",
                     fieldLabel: t('navigation'),
                     name: 'name',
-                    width: "100%"
+                    width: "100%",
```

Comment:
Please remove the trailing `,` here, that's unfortunately not valid JS. 
It works in Chrome and FF, but anyway, we should have everything clean. ;)

Reply:
@brusch Just FYI, that is valid JavaScript (ES5) and also reduces git diff noise. Airbnb for example has it in their guidelines by default: https://github.com/airbnb/javascript#commas ;)

Reply:
that's my linter config (airbnb)... but it's up to @brusch to define his / their code styles so I apply to his rules :)